examples/bri-3/src/bri/vsm/capabilites/executeVsmCycle/executeVsmCycleCommand.handler.ts

```diff
@@ -4,7 +4,7 @@ import { TransactionAgent } from '../../../transactions/agents/transactions.agen
 import { TransactionStatus } from '../../../transactions/models/transactionStatus.enum';
 import { WorkstepStorageAgent } from '../../../workgroup/worksteps/agents/workstepsStorage.agent';
 import { ExecuteVsmCycleCommand } from './executeVsmCycle.command';
-import { WorkstepExecutionFailuresEvent } from '../handleWorkstepFailuresEvents/workstepExecutionFailures.event';
+import { WorkstepExecutionEvent } from '../handleWorkstepEvents/workstepExecution.event';
```

Comment:
i think we can adopt naming for events in past tense, as they mark that something has  happened:

WorkstepExecutionEvent ->  WorkstepExecutedEvent

Reply:
@ognjenkurtic Done. Thank you for reviewing!